ReactiveCocoa/Swift/Signal.swift

```diff
@@ -181,14 +181,24 @@ extension SignalType {
 		return observe(Observer(action))
 	}
 
-	/// Observes the Signal by invoking the given callback when `next` events are
-	/// received.
+	@available(*, unavailable, message="This Signal may emit errors which must be handled explicitly, or observed using observeResult:")
```

Comment:
I think this and `startWithNext` should be marked as `deprecated`, not `unavailable`, since this is targeting RAC 4.

Also, the message should probably say `observeResult()`, not `observeResult:`. ☺️